packages/base/spec.gts

```diff
@@ -126,9 +136,16 @@ class Isolated extends Component<typeof Spec> {
           <h2 id='examples'>Examples</h2>
         </header>
         {{#if (eq @model.specType 'field')}}
-          <@fields.containedExamples @typeConstraint={{this.absoluteRef}} />
-        {{else}}
-          <@fields.linkedExamples @typeConstraint={{this.absoluteRef}} />
+          {{#if this.isPrimitiveField}}
+            <p
+              class='spec-example-incompatible-message'
+              data-test-spec-example-incompatible-primitives
+            >
+              <span>Spec cannot render examples for primitive fields</span>
+            </p>
```

Comment:
There's a better language in ticket 7929: For primitive fields, display message "Examples are not currently supported for primitive fields."

Reply:
Oh Yea. I forgot about that. Thanks. I will resolve